CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -447,6 +470,7 @@ local function MakeButton(name, text, size, clickFunc, pageRef, hubRef)
 end
 
 local function CreateDropDown(dropDownStringTable, startPosition, settingsHub)
+	local Panel3D = require(CoreGui.RobloxGui.Modules.VR.Panel3D)
```

Comment:
this feels weird to require for every drop down, even if we aren't a 3d renderer. Could we just use it if we need it?